src/base.scss

```diff
@@ -7,8 +7,6 @@ html {
         width: 100%;
         height: 100%;
         margin: 0;
-        font-family: 'Source Sans Pro', sans-serif;
```

Comment:
Why are we removing this? We will have to assign a style everywhere now... If you are unable to override this font decrease specificity here or increase specificity in `SearchBox.scss`.

My suggestion would be to decrease the specificity here and extract all styles to there individual blocks (basically un-nest them) so their specificity is based off of one tag at a time.